### PR TITLE
Tweaks for 1995 front end

### PR DESCRIPTION
**Pull request by jbalboni**
These are the schema changes I made for the 1995 front end. Some of them affect common components. I don't think they'll have a significant impact on the backend, but let me know if they do.

Changes:

- Removed `present` as option for end of date range. We don't ever use this.
- Removed `salutation` from name. Also not ever used.
- Added a pattern for file number, since we're validating this on the front end.
- Added `type` to enum fields, since this is required for the front end.
- Changed the order of some fields/enums because it affects the front end.
- Added preferredContactMethod field (same as 1990 form).

### CODE REVIEW

File: dist/edu-benefits-schema.json [hunk 1]
```diff
@@ -216,9 +216,6 @@
     "fullName": {
       "type": "object",
       "properties": {
-        "salutation": {
-          "type": "string"
-        },
```
Comment (lihanli) [inline dist/edu-benefits-schema.json]: is this not used in the 1990?

Comment (jbalboni) [inline dist/edu-benefits-schema.json]: No, our name component used in hca and the 1990 doesn't have this field.

File: src/change-of-program/schema.js [hunk 2]
```diff
@@ -18,7 +18,8 @@ export default {
     'serviceBefore1977',
     'date',
     'dateRange',
-    'educationType'
+    'educationType',
+    'preferredContactMethod',
```
Comment (lihanli) [inline src/change-of-program/schema.js]: trailing comma

File: dist/change-of-program-schema.json [hunk 3]
```diff
@@ -338,6 +326,14 @@
...
         "cooperativeTraining"
       ]
+    },
+    "preferredContactMethod": {
+      "type": "string",
+      "enum": [
+        "mail",
+        "email",
+        "phone"
+      ]
```
Comment (lihanli) [inline dist/change-of-program-schema.json]: can you add this definition to the 1990 schema?

Comment (jbalboni) [inline dist/change-of-program-schema.json]: Done.

**Pull request merged**